packages/legend-studio/style/components/editor/aux-panel/_execution-plan-viewer.scss

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Don't think the aux panel is the proper place for this file. maybe under the element editor css files is better